integration/tools/benchmark/pytorch/Dockerfile

```diff
@@ -0,0 +1,14 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+from pytorch/pytorch:1.9.0-cuda10.2-cudnn7-runtime
+
+RUN pip install alluxio prometheus_client
```

Comment:
This is an independent container for pytorch?

Reply:
Yeah, mainly because we use the prometheus_client.
Probably i can remove this dependency, so that we can directly use pytorch docker image, but the code maybe more ugly